src/ontology/mondo-edit.obo

```diff
@@ -309897,7 +309897,7 @@ subset: ordo_malformation_syndrome {source="Orphanet:280576"}
 subset: orphanet_rare {source="Orphanet:280576"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
-synonym: "BANF1-related neurodevelopmental syndrome" EXACT CLINGEN_LABEL [https://search.clinicalgenome.org/CCID:008387]
+synonym: "BANF1-related neurodevelopmental syndrome" EXACT CLINGEN_LABEL [https://search.clinicalgenome.org/CCID\:008387]
```

Comment:
I did not make this change. I am not sure why this is showing up since I pulled `master` before creating the branch for this work.

Reply:
I will remove it from the PR